Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -78,6 +78,7 @@ def get_tested_integrations_for_collected_tests(self, collected_tests):
 
         return tested_integrations
 
+    # This function is the same function exactly as 'get_content_pack_name_of_test' and therefore should be removed
```

Comment:
lets open an issue for this and add TODO here

Reply:
https://github.com/demisto/etc/issues/30672